src/Plugin/EncryptionMethod/McryptAES256Encryption.php

```diff
@@ -12,7 +12,8 @@
  * @EncryptionMethod(
  *   id = "mcrypt_aes_256",
  *   title = @Translation("Mcrypt AES 256"),
- *   description = "This uses PHPs mcrypt extension and <a href='http://en.wikipedia.org/wiki/Advanced_Encryption_Standard'>AES-256</a>."
+ *   description = "This uses PHPs mcrypt extension and <a href='http://en.wikipedia.org/wiki/Advanced_Encryption_Standard'>AES-256</a>.",
```

Comment:
We're abandoning it. Per @rlhawk its received no love for years and requires a separate PHP library in lieu of a composer library or some equivalent.